### ISSUE TITLE
Dropdown focus announcement bug in Safari / VoiceOver

### ISSUE THREAD

**Issue by origami-z** [jpmorganchase/salt-ds#3490]
https://storybook.saltdesignsystem.com/iframe?args=&id=core-dropdown--with-form-field&viewMode=story

<img width="647" alt="image" src="https://github.com/jpmorganchase/salt-ds/assets/5257855/5cc83735-3784-4bbd-800d-a9d3c2d42783">

- When moving between options, nothing is announced
- "Completion" selected, not sure where does it come from

**Comment by origami-z** [comment]
Maybe related to https://github.com/select2/select2/issues/4349

**Comment by origami-z** [comment]
This is a Safari bug, given VoiceOver + Chrome worked

**Comment by joshwooding** [comment]
Definitely a webkit bug https://bugs.webkit.org/show_bug.cgi?id=231724

@jake-costa this is what I mentioned the other day

**Comment by jake-costa** [comment]
@origami-z @joshwooding Great call out. As Josh mentioned, this is a known issue for VoiceOver+Safari. Thoughts on adding to the a11y section on the site for awareness?

Resources:
https://discussions.apple.com/thread/255614236?sortBy=rank 
https://github.com/w3c/aria-practices/issues/2060

**Comment by joshwooding** [comment]
@jake-costa we should definitely add it, could you draft something?

**Comment by origami-z** [comment]
Tested again today, with MacOS 14.7 (23H124) and Safari 18.0.1, no change to the bug. Navigation between items doesn't trigger announcement, also "completion selected" is still there

**Comment by joshwooding** [comment]
Can we check it this is an issue with Safari 18? I've read reports that it has been fixed.

**Comment by origami-z** [comment]
> Can we check it this is an issue with Safari 18? I've read reports that it has been fixed.

I was testing Safari 18.0.1

**Comment by joshwooding** [comment]
> > Can we check it this is an issue with Safari 18? I've read reports that it has been fixed.
> 
> I was testing Safari 18.0.1

Weird... maybe it was 18.1

**Comment by origami-z** [comment]
+1 SO 92916

**Comment by joshwooding** [comment]
@jake-costa I can confirm this is no longer a problem in the latest version of Safari + MacOS but we should still document it as it will affect lots of users.

**Comment by jake-costa** [comment]
@joshwooding I have Safari 18.1 but am unable to update my Mac to 15.1 (Currently on 14.7.1) to validate. That said, since you are confirming it is no longer an issue on 15.1, we can modify the documentation to say

Issue Details:
OS: MacOS 14.X.X or Below MacOS 15.

Thoughts?
